go.mod

```diff
@@ -292,6 +292,8 @@ replace github.com/apache/thrift => github.com/apache/thrift v0.14.1
 
 replace github.com/hashicorp/consul => github.com/hashicorp/consul v1.10.2
 
+replace github.com/tidwall/match => github.com/tidwall/match v1.1.1
```

Comment:
Add a comment for when to remove the replace directive?

Reply:
Done!